content/blog/peace-of-mind-with-cloud-secret-providers/index.md

```diff
@@ -0,0 +1,669 @@
+---
+title: "Peace of Mind with Cloud Secret Providers"
+
+date: 2020-05-07T14:30:35-07:00
+draft: false
+meta_desc: "Encrypt your Pulumi secrets with a cloud secrets provider"
+meta_image: meta.png
+authors:
+    - lee-briggs
+tags:
+    - features
+    - security
+---
+
+The secrets in your infrastructure are a vital part of your security model, and provisioning infrastructure is an inherently privileged process. [Previously](https://www.pulumi.com/blog/managing-secrets-with-pulumi/) we introduced client-side secret encryption and started encrypting secret configuration values inside the Pulumi state so that users could be confident their passwords, tokens and other secret values were viewable only by them while managing their infrastructure.
+Our first iteration of the client-side encryption used a passphrase for encrypting the secret, however, remembering that passphrase and sharing it securely with team members is less than ideal.
+That's why we also added support for "Cloud Secret Providers", giving users full confidence that their sensitive values are for their eyes only.
```

Comment:
I wonder if we should update our docs to also use this term Cloud Secret Providers.  The language we currently use there is a bit different (but probably not better): https://www.pulumi.com/docs/intro/concepts/config/#initializing-a-stack-with-alternative-encryption

Reply:
I can do this as a separate item!